tests/mgbench/benchmark.py

```diff
@@ -408,7 +408,7 @@ def get_query_cache_count(
     vendor_specific_args = helpers.parse_kwargs(args.vendor_specific)
 
     assert args.benchmarks != None, helpers.list_available_workloads()
-    assert args.vendor_name == "memgraph" or args.vendor_name == "neo4j", "Unsupported vendors"
+    # assert args.vendor_name == "memgraph" or args.vendor_name == "neo4j" or args.vendor_name == "memgraphdocker", "Unsupported vendors"
```

Comment:
Isn't this redundant because of the `choices` under args?

Reply:
This has been removed, and sub-arguments have been introduced.